app/helpers/application_helper.rb

```diff
@@ -18,6 +18,12 @@ def status_subject status
     status ? "list-group-item-success" : "list-group-item-light"
   end
 
+  def status_course course
+    return "joined" if course.ready?
```

Comment:
như này nên dùng case when

Reply:
Case when này có nhiều cách quá. Cái em vừa push lên kia quên mất chưa check lại rubocop.
Nó warning rồi " Do not use empty case condition, instead use an if expression."